types/relay-runtime/lib/util/RelayConcreteNode.d.ts

```diff
@@ -14,6 +15,13 @@ export interface ConcreteRequest {
     readonly params: RequestParameters;
 }
 
+// Note: the phantom type parameter here helps ensures that the
+// $Parameters.js value matches the type param provided to preloadQuery.
+export interface PreloadableConcreteRequest<TQuery extends OperationType> {
+    kind: "PreloadableConcreteRequest";
```

Comment:
Why, because of an older TypeScript version? The `kind` should only allow that specific string literal and it has already (correctly) been that way in the `react-relay` package.

Reply:
Yeah I know, but I'm not sure of the consequences. OSS has never had this supported, so we don't know what will break. All the other request types has the kind set to string for reasons i cannot recall right now.

Reply:
Got it! Aligned it to the other request types in 28d0f0c1a56fc69a37c652a5224a3aac9a9eee81